curriculum/challenges/english/01-responsive-web-design/css-grid-magazine/part-024.md

```diff
@@ -0,0 +1,244 @@
+---
+id: 6143d2842b497779bad947de
+title: Part 24
+challengeType: 0
+dashedName: part-24
+---
+
+# --description--
+
+To start your CSS, normalize the CSS rules by targeting all elements with `*`, including the `::before` and `::after` psuedo-selectors. Set the `padding` property and `margin` property both to `0`.
```

Comment:
```suggestion
To start your CSS, normalize the CSS rules by targeting all elements with `*`, including the `::before` and `::after` pseudo-selectors. Set the `padding` property and `margin` property both to `0`.
```

Reply:
I can spell I promise 🤦🏻

Reply:
Should have stuck with my `sudo-elements`...